config/config.go

```diff
@@ -0,0 +1,35 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/jinzhu/configor"
+)
+
+func FromJSON(config interface{}, name string) error {
+
+	// we need to get the absolute path and also test it because of the
+	// difference of how this code runs in different environments
+	absPath, _ := filepath.Abs(filepath.Join("./_config/", name))
+
+	if _, err := os.Stat(absPath); os.IsNotExist(err) {
```

Comment:
What happens if there is any other error than IsNotExist? We should probably fail here out of the function.